ma_serial:
  port: "/dev/ttyACM0"
  baud_rate: 115200
  topic: "cmd_vel"
  time_out: 1000
  debug: true
  robot_id: 7
  red_outpost_HP: 300
  blue_outpost_HP: 100
  game_time: 3